dts/arm64/nxp/nxp_mimx95_a55.dtsi

```diff
@@ -239,4 +239,70 @@
 		#mbox-cells = <1>;
 		status = "disabled";
 	};
+
+	tpm1: tpm@44310000 {
```

Comment:
would you please follow the address sequence to align the nodes? thanks.

Reply:
It's more readable putting the same IP block nodes together. In Linux it ordered the nodes by the unit address giving a bad experience.

Reply:
okay.